scripts/build_native_deps.sh

```diff
@@ -325,6 +339,11 @@ build_pixman()
     local build="$BUILD/$PIXMAN_DIR"
 
     extract "$PIXMAN_ARCHIVE" "$src"
+    if [ "$uname" = "Darwin" ]; then
+        sed_i -E \
+            "/^(scaling_helpers|region)_test_OBJECTS.*$/ s/$/ utils.\$(OBJEXT)/" \
+            "$src/test/Makefile.in"
+    fi
```

Comment:
I'd prefer to just not build tests and demos all along:
```sh
sed_i "s|SUBDIRS = pixman demos test|SUBDIRS = pixman|" "$src/Makefile.in"
```

Reply:
Fair enough. Will fix.